posts/2023-05-30-23.0.0.5.adoc

```diff
@@ -0,0 +1,183 @@
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/24577[Memory Leak in MicroProfile OpenAPI's SchemaRegistry.current]
++
+A user reported a memory leak that occurred with each application restart, where 100MB of additional memory was used each time.  The culprit ended up coming from MicroProfile OpenAPI's `SchemaRegistry` class.
++
+This issue has been reported upstream to SmallRye, and has also been fixed directly in Liberty.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/24864[HTTP/2 max frame size exceeded when compression is used]
++
+When compression is configured in the `server.xml` on an `httpendpoint` and `http/2` is used, the `http/2` max frame size may be exceeded, leading to a `FRAME_SIZE_ERROR` appearing in the server's log.
++
+This issue has been resolved and the `http/2` response data is now split into multiple data frames to avoid sending a data frame larger than the `http/2` max frame size of the client.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/25010[EntryNotFoundException thrown in federated registries when using custom input/output configuration]
++
+When running with `federatedRegistries-1.0`, it is possible to get an `EntryNotFoundException` when defining a non-identifier type property for the federated registries input/output mapping. This exception can occur in any of the `com.ibm.ws.security.wim.registry.util.*Bridge` classes, but the key is it originates from a `BridgeUtils.getEntityByIdentifier` call.
++
+The following is an example stack:
++
+[source]
+----
+com.ibm.websphere.security.EntryNotFoundException: CWIML1010E: The user registry operation could not be completed. The uniqueId = null and uniqueName = null attributes of the identifier object are either not valid or not defined in the back-end repository.
+	at com.ibm.ws.security.registry.internal.UserRegistryWrapper.getGroupsForUser(UserRegistryWrapper.java:248)
+	at web.UserRegistryServlet.handleMethodRequest(UserRegistryServlet.java:140)
+	at web.UserRegistryServlet.doGet(UserRegistryServlet.java:174)
+	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
+	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
+	at com.ibm.ws.webcontainer.servlet.ServletWrapper.service(ServletWrapper.java:1258)
+	... 
+----
++
+This issue has been resolved and the method no longer throws `EntryNotFoundException`.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/24939[`requestTiming-1.0` causes elevated (or spiking) CPU performance due to the `SlowRequestManager` ]
++
+When using the `requestTiming-1.0` feature in OpenLiberty, the CPU usage is elevated. CPU impact correlates to CPU capacity. 
++
+This is more obvious when a lower threshold is set for the "slow request" threshold (i.e. e.g. <= 15s). Even so, may not encounter a noticeable impact depending on CPU capacity. 
```

Comment:
I also spotted that the less than or equals sign `<=` is getting rendered as an arrow. Pointing that out to see if we want to fix the misrendering. However I understand this `<=` may have been lifted from some kind of text scrapping from the original GitHub issue.

<img width="1052" alt="image" src="https://github.com/OpenLiberty/blogs/assets/31117513/76e17e92-741b-4a67-84a0-4588661aef6e">

Reply:
Resolved with https://github.com/OpenLiberty/blogs/pull/3101/commits/91e93834e1c6d3c590f8c6e6abf714db932668da